interface/patient_file/summary/add_edit_issue.php

```diff
@@ -375,6 +375,7 @@ function ActiveIssueCodeRecycleFn($thispid2, $ISSUE_TYPES2) {
 <html>
 <head>
 <?php html_header_show();?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-9-1/index.js"></script>
```

Comment:
It will not help since in line 594 you will override the validate.js function validate and it will go to the old and not new validation we wrote. And to put it underneath somewhere  will make it go out of syntax block . so I think the best place to leave it is there .